src/lib/button-toggle/button-toggle.scss

```diff
@@ -47,3 +53,27 @@ $mat-button-toggle-border-radius: 2px !default;
 .mat-button-toggle-label-content > * {
   vertical-align: middle;
 }
+
+// Overlay to be used as a tint. Note that the same effect can be achieved by using a pseudo
+// element, however we use a proper DOM element in order to be able to disable the default
+// touch action. This makes the buttons more responsive on touch devices.
+.mat-button-toggle-focus-overlay {
+  // The button spec calls for focus on raised buttons (and FABs) to be indicated with a
+  // black, 12% opacity shade over the normal color (for both light and dark themes).
+  background-color: rgba(black, 0.12);
```

Comment:
I think only theme related should be in theme file. Default is black which will not change with the theme.

Added theme color in the theme file.